crates/ruma-common/tests/api/header_override.rs

```diff
@@ -1,7 +1,8 @@
 #![allow(clippy::exhaustive_structs)]
+#![cfg(feature = "api")]
```

Comment:
Files in subdirectories of `tests` won't be included in the test suite automatically. I think having the subdirectory makes sense, but to make it work, you need to create `tests/api.rs` that has `mod` statements for the `api/*` files. Then you just need a single `cfg` too.

Reply:
Oh, hm. Well I guess that works...